src/app/persistence/PascalString.h

```diff
@@ -162,6 +171,10 @@ class PascalBuffer
         return len == kInvalidLength || (static_cast<size_t>(len + PREFIX_LEN) <= span.size());
     }
 
+    /// Is the buffer that the pascal string points into a valid
+    /// pascal string (null or valid length?)
+    bool IsValidContent() const { return IsValid({ mData, static_cast<size_t>(mMaxSize) + PREFIX_LEN }); }
```

Comment:
1. The naming here is odd, because earlier in this API Content refers to the actual data, not including the length.  Should this just be called HasValidLength?
2. It's a bit unfortunate that we can't use ContentWithLenPrefix() and have to copy it.  But could we just make `IsValid` take a `ByteSpan`, not a `Span<const T>`?  This looks like the first non-test use of `IsValid` (for some values of "non-test", since IsValidContent is only used in tests), and all the tests that use `IsValid` so far have `T` as `uint8_t`.

Reply:
I think I can drop this one maybe that is easier. I used to need this in previous API versions, however I do not use it now execept unit tests. Will double-check.